### PR TITLE
move to cri-o/cri-o and build with containers/conmon

### PR DESCRIPTION
**Pull request by haircommander**
Signed-off-by: Peter Hunt <pehunt@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
Move doesn't seem complete, so I'll mark this WIP for now - feel free to un-WIP once the repo changes locations officially

File: contrib/cirrus/lib.sh [hunk 1]
```diff
@@ -312,9 +312,9 @@ install_conmon(){
         GOPATH $GOPATH
         CRIO_COMMIT $CRIO_COMMIT
     "
-    DEST="$GOPATH/src/github.com/kubernetes-sigs/cri-o.git"
+    DEST="$GOPATH/src/github.com/cri-o/cri-o.git"
     rm -rf "$DEST"
-    ooe.sh git clone https://github.com/kubernetes-sigs/cri-o.git "$DEST"
+    ooe.sh git clone https://github.com/cri-o/cri-o.git "$DEST"
```
Comment (mheon) [inline contrib/cirrus/lib.sh]: Testing testing testing

Comment (mheon) [reply contrib/cirrus/lib.sh]: Alright, I can add comments here, but not on other files in the diff.
What the hell, Github.

File: contrib/cirrus/packer/prior_fedora_base-setup.sh [hunk 2]
```diff
@@ -1 +0,0 @@
-fedora_base-setup.sh
```
Comment (mheon) [inline contrib/cirrus/packer/prior_fedora_base-setup.sh]: What happened here? File mixup? @cevich

**Comment by mheon** [comment]
Alright, I can see cri-o/cri-o, but Github is really, really unhappy about the fact that it moved, and the main page doesn't want to load yet.

Let's give this a few hours to settle, then.

File: contrib/cirrus/packer/prior_fedora_base-setup.sh (hunk 2 above)
Comment (cevich) [reply contrib/cirrus/packer/prior_fedora_base-setup.sh]: Whoa, wha?  This file should be a symlink to `fedora_base-setup.sh`

**Comment by mheon** [comment]
Alright, I think this has settled enough. Removing WIP.

**Comment by mheon** [comment]
LGTM once tests go green
@rhatdan @baude @vrothberg @giuseppe @TomSweeneyRedHat PTAL

**Comment by mheon** [comment]
Actually, un-LGTM - we should figure out with @cevich what happened with the symlinked file

**Comment by mheon** [comment]
@haircommander I think the images failure might be legitimate... @baude that means we broke RPM builds, right?

**Comment by cevich** [comment]
@mheon that file should be a symlink as currently the F28 and F29 VM images build exactly the same.  When F30 officially arrives, that could change.  My best guess is a simple accident caused it to change into a file.

**Comment by cevich** [comment]
Oh and BTW, this PR inspired me to do https://github.com/containers/conmon/pull/15 which is all-green also :smile:

**Comment by haircommander** [comment]
@cevich is there action I need to take on the symlink thing?

**Comment by cevich** [comment]
@haircommander yes please...just put the symlink back :smile: 

``
cd contrib/cirrus/packer
ln -sf fedora_base-setup.sh prior_fedora_base-setup.sh
git add prior_fedora_base-setup.sh
``

Thanks for checking.

**Comment by mheon** [comment]
@haircommander Needs a rebase to pick up Lint fixes

**Comment by mheon** [comment]
Image job failing consistently here, looks like?

**Comment by haircommander** [comment]
@mheon yeah it is. the error is confusing to me but I'll look into it

**Comment by haircommander** [comment]
Ah shoot yeah the AMI has to be changed for ci/prow/images. Let me see if I have permission to do so

**Comment by haircommander** [comment]
Update, I can't find them. I guess we'll have to wait until @cevich is back :\

**Comment by mheon** [comment]
@baude You're the closest thing we have to an expert on the Prow stuff - any idea what we need to do here, or are we tagging in Steve to figure it out?

**Comment by haircommander** [comment]
@cevich are you able to change the AMI to not use kubernetes-sigs directory anymore?

**Comment by mheon** [comment]
These aren't AMIs, they're container images - we're on the newer Prow CI system. I don't know where the Dockerfiles live though - presumably somewhere hosted by Prow.

**Comment by cevich** [comment]
Oh gosh, I spent a day reading through Prow and OpenShift/release docs and readmes.  My ears began bleeding brain matter.  However, this is more than likely due to my lack of OpenShift/Kube expertise.  [This may make much more sense to someone else.](https://github.com/openshift/release/tree/master/ci-operator)

**Comment by haircommander** [comment]
Ideally we could land https://github.com/containers/conmon/pull/20 to have the latest and greatest conmon before merging this

**Comment by mheon** [comment]
Works for me

**Pull request merged**